package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@banno/node-vault-fork",
-  "version": "0.9.2",
+  "version": "0.9.4",
```

Comment:
Please make sure to add git tags for these versions. `npm version` should be doing that automatically.
Adding the release notes to GH releases is recommended too. (This is not yet automated.)